tests/fixtures/bugs.json

```diff
@@ -32,3 +32,9 @@
8, "creation_time": "2017-06-12T20:24:38Z", "author": "mozillamarcia.knous@gmail.com"}, {"count": 3, "text": "Created attachment 8879587\nCore dump of Firefox GPU process\n\nI beg your pardon that I generated so many crash reports (between 90 and 100). I will disable the GPU process after I have sent this message.\n\nTo help you out I compiled a nightly Firefox with debug symbols. Today I was finally lucky to catch a core dump of the crashing GPU process. The attached file has information about the core dump, a backtrace and a full backtrace. If it helps, I can try to upload the complete core dump to bugzilla (4.9M, lz4 compressed).\n\nHere\u2019s the crash report that was automatically generated: https://crash-stats.mozilla.com/report/index/43be6b3c-1d43-4981-b447-ade511170620\n\nHow to reproduce:\n * Install Fedora 26\n * Download and unpack Firefox nightly\n * Start Firefox\n * Force-enable the GPU process\n * Restart Firefox\n\nThe GPU process crashes within the first 30 seconds which is not visible for the user. The browser works as expected, it just generates crash reports. It doesn\u2019t happen on every browser start.\n\nJust ask if you need any information.", "id": 12414731, "creation_time": "2017-06-20T15:34:24Z", "author": "marcus.husar@gmail.com"}, {"count": 4, "text": "Hello, I am the one with the most [@ libX11.so.6.3.0@0x4259b ] crashes.\n\nThe cause: I opened a pdf with pdfium and network.disable.ipc.security;false\n\nFor example:\nbp-11665dc6-7488-4ee8-903f-231b40170615 15.06.17 11:20 [@ libX11.so.6.3.0@0x4259b ]\n= this bug\nbp-0a1275b3-d959-40cf-9080-07caa0170615 15.06.17 11:20 [@ mozilla::net::NeckoParent::GetValidatedOriginAttributes ]\n= bug 1322610\n\n\nbp-f4b851c9-c940-4e2e-8058-622e30170609 09.06.17 17:50 [@ libX11.so.6.3.0@0x4259b ]\n= this bug\nbp-5a09146a-9cb4-40e3-b831-04d5e0170609 09.06.17 17:50 [@ mozilla::net::NeckoParent::GetValidatedOriginAttributes ]\n= bug 1322610\n\n\n(Darkspirit from bug 1322610 comment #9)\n> network.disable.ipc.security;false (default=true)\n> \n> https://gopherproxy.meulie.net/ygrex.ru/h/debian/firefox-security.html\n> > Interprocess security checks (disabled by default per bug 820712):\n> > network.disable.ipc.security=false\n> \n> I just thought that \"disabling\" \"security\" is needed for some old addons or\n> plugins, so I set it to false. \"Security checks\" sounded good.", "id": 12414897, "creation_time": "2017-06-20T16:21:41Z", "author": "jan@ikenmeyer.eu"}, {"count": 5, "text": "(In reply to Darkspirit from comment #4)\n> pdfium\nNo, sorry, I meant pdfjs.", "id": 12414910, "creation_time": "2017-06-20T16:24:40Z", "author": "jan@ikenmeyer.eu"}, {"count": 6, "text": "No user should change network.disable.ipc.security and it will get deleted in the future (post-57). See bug 1322610 comment 6.", "id": 12414919, "creation_time": "2017-06-20T16:27:45Z", "author": "jan@ikenmeyer.eu"}, {"count": 7, "text": "(In reply to Marcus Husar from comment #3)\n> How to reproduce:\n>  * Install Fedo
```

Comment:
Are you sure you actually need to add more bugs? The first bugs in the list might already have an assignee

Reply:
Ah you need it because of MINIMUM_ASSIGNMENTS, but you could just overwrite it during the test to be 1.

Reply:
I can do that. Is failing the test is related to these added bugs?

Reply:
Yes, indeed they are passing now :)